umbraco-cms/fundamentals/setup/upgrading/README.md

```diff
@@ -1,16 +1,132 @@
+
+1. Stop your site in IIS to prevent any changes being while you are upgrading.
+2. Open your Umbraco project in Visual Studio.
+3. Right-click on the project name in the Solution Explorer and select **Properties**.
+4. Select the **.NET** version from the **Target Framework** drop-down.
+5. Go to **Tools** > **NuGet Package Manager** > **Manage NuGet Packages for Solution...**
+6. Go to the **Installed** tab in the NuGet Package manager.
+7. Choose **Umbraco.Cms**.
+8. Select thecorrect version from the **Version** drop-down.
+9. Click **Install** to upgrade your project.
+10. Consult the [version specific upgrade notes for Umbraco Forms](../../../../umbraco-forms/installation/version-specific.md) if relevant.
+11. Restart your site in IIS, build and run your project to finish the installation.
+
+## Upgrade to a new Minor
+
+NuGet installs the latest version of the package when you use the `dotnet add package` command unless you specify a package version:
+
+`dotnet add package Umbraco.Cms --version <VERSION>`
+
+Add a package reference to your project by executing the `dotnet add package Umbraco.Cms` command in the directory that contains your project file.
+
+Run `dotnet restore` to install the package.
+
+{% hint style="warning" %}
+If you are using SQL CE in your project you will need to run `dotnet add package Umbraco.Cms.SqlCe --version <VERSION>` too before running the `dotnet restore` command.
+{% endhint %}
+
+When the command completes, open the **.csproj** file to make sure the package reference was updated:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Umbraco.Cms" Version="x.x.x" />
+</ItemGroup>
+```
+
+## Run an unattended upgrade
+
+When upgrading your Umbraco project it is possible to enable the upgrade to run unattended. This means that you will not need to run through the installation wizard when upgrading.
+
+Below you will find the steps you need to take in order to upgrade your project unattended.
+
+{% hint style="info" %}
+Are you running a load balanced setup with multiple servers and environments?
+
+Check out the section about [Unattended upgrades in a load balanced setup](#unattended-upgrades-in-a-load-balanced-setup).
+{% endhint %}
+
+### Enable the unattended upgrade feature
+
+1. Add the `Umbraco:Cms:Unattended:UpgradeUnattended` configuration key.
+2. Set the value of the key to `true`.
+
+{% code title="appsettings.json" %}
+```json
+{
+    "Umbraco": {
+        "CMS": {
+            "Unattended": {
+                "UpgradeUnattended": true
+            }
+        }
+    }
+}
+```
+{% endcode %}
+
+### Run the upgrade
+
+With the correct configuration applied, the project will be upgraded on the next boot.
+
+#### Boot order
+
+The Runtime level will use `Run` instead of `Upgrade` in order to allow the website to continue to boot up directly after the migration is run. This happens instead of initiating the otherwise required restart.
```

Comment:
```suggestion
The Runtime level will use `Run` instead of `Upgrade` to allow the website to continue to boot up directly after the migration is run. This happens instead of initiating the otherwise required restart.
```